src/main/java/com/gooddata/md/visualization/VisualizationObject.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) 2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+
+package com.gooddata.md.visualization;
+
+import com.fasterxml.jackson.annotation.*;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+import static com.gooddata.md.visualization.CollectionType.*;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.Validate.notNull;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeType;
+import com.gooddata.executeafm.UriObjQualifier;
+import com.gooddata.executeafm.afm.*;
+import com.gooddata.md.AbstractObj;
+import com.gooddata.md.Meta;
+import com.gooddata.md.Queryable;
+import com.gooddata.md.Updatable;
+
+import java.io.Serializable;
+import java.util.*;
+
+@JsonTypeName(VisualizationObject.NAME)
+@JsonTypeInfo(include = WRAPPER_OBJECT, use = NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class VisualizationObject extends AbstractObj implements Queryable, Updatable {
+    static final String NAME = "visualizationObject";
+    private Content content;
+
+    @JsonCreator
+    public VisualizationObject(@JsonProperty("content") final Content content, @JsonProperty("meta") final Meta meta) {
+        super(meta);
+        this.content = content;
+    }
+
+    public Content getContent() {
+        return content;
+    }
+
+    @JsonIgnore
+    public List<Bucket> getBuckets() {
+        return getContent().getBuckets();
+    }
+
+    @JsonIgnore
+    public List<Measure> getMeasures() {
+        return getContent().getMeasures();
+    }
+
+    @JsonIgnore
+    public List<Measure> getSimpleMeasures() {
+        return getMeasures().stream()
+                .filter(measure -> measure.getDefinition() instanceof VOSimpleMeasureDefinition)
```

Comment:
Can make it `VOSimpleMeasureDefinition.class:isInstance`

Reply:
Not really I think. I'm checking definition from measure, not measure directly. Maybe you can advice more fancy syntax for it? :)

Reply:
Right, I didn't notice... you could do this though: `getMeasures().stream().map(Measure::getDefinition).filter(VOSimpleMeasureDefinition.class:isInstance)`